src/Hyperlink/index.jsx

```diff
@@ -23,14 +24,9 @@ function Hyperlink(props) {
     other.rel = other.rel ? `noopener ${other.rel}` : 'noopener';
 
     externalLinkIcon = (
-      // Space between content and icon
-      <span>{' '}
-        <span
-          className={classNames('fa', 'fa-external-link')}
-          aria-hidden={false}
-          aria-label={externalLinkAlternativeText}
-          title={externalLinkTitle}
-        />
+    // Space between content and icon
+      <span className="d-inline-block">{' '}
+        <Icon src={Launch} />
```

Comment:
```suggestion
        <Icon src={Launch} style={{ height: '1em', width: '1em' }} />
```

Let's make the size of the icon match the size of the text